proposals/Modules.md

```diff
@@ -0,0 +1,65 @@
+## Modules
+
+Modules will provide the basic organization building block in Serulian.
+
+### Basic Definition
+
+A module is defined as a single Serulian file (ending in `.seru`) that **exports** some set of types
```

Comment:
Open Question: Should we support some type of directory-level module or are using files (that, in turn, import other files) work?